src/vs/workbench/contrib/notebook/test/browser/notebookExecutionService.test.ts

```diff
@@ -36,6 +36,10 @@ suite('NotebookExecutionService', () => {
 	let kernelService: INotebookKernelService;
 	let disposables: DisposableStore;
 
+	teardown(() => {
```

Comment:
Instead of disposables as a separate variable, `ensureNoDisposablesAreLeakedInTestSuite` returns a disposable store to use.

Reply:
`ensureNoDisposablesAreLeakedInTestSuite` isn't being called in this test yet. There's still a leak that @rebornix says he'll look into